dev-scripts/port-program.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+PROGRAM_FILE=$1
+KEYPAIR_PATH=$2
+
+echo "deploy program file path: $PROGRAM_FILE"
+echo "keypair path: $KEYPAIR_PATH"
+
+solana program deploy $PROGRAM_FILE --keypair $KEYPAIR_PATH --url $CLUSTER_URL
```

Comment:
a thêm flag: `--program-id` vào phòng trường hợp developer muốn deploy vào 1 địa chỉ cố định nhé 

`solana program deploy --help` for more info

Reply:
`--program-id` chỉ dùng cho case upgrade program thôi, còn khi mình port là lần đâu deploy, nên ko cần thêm option này đâu. Nếu thêm option này thì mình viết script riêng dùng chung, chứ ko riêng port dapp @kientt6034

Reply:
@minhdc0703 trong anchor `lib.rs` nó có phần `declare_id`. Liệu có bị conflict gì với phần này ko a?